src/ui/LiveSplit.tsx

```diff
@@ -100,7 +100,7 @@ export class LiveSplit extends React.Component<Props, State> {
                 }
             }
         } catch (_) { /* Looks like the storage has no valid data */ }
-        if (hotkeySystem == null) {
+        if (hotkeySystem === null) {
```

Comment:
I don't want this. `== null` / `!= null` has its place (for checking against both null and undefined) and in fact my local tslint is totally fine with it (unlike == in other cases). So can we please change the linter settings here to allow it in those cases? (I haven't checked if this particular case even could have both null and undefined. If it doesn't, then I'm possibly fine with the change)